### ISSUE TITLE
Updating to v3.2.1 issues a warning about node version required

### ISSUE THREAD

**Issue by idcrook** [bwp91/homebridge-platform-wemo#263]
### What issue do you have? Please be as thorough and explicit as possible.

Updating to v3.2.1 issues a warning about node version

https://github.com/bwp91/homebridge-platform-wemo/releases/tag/v3.2.1

Update in Homebridge UI-X complains about node version (says it requires `14.17.3`)

### Details of your setup.
* (1) Homebridge UI-X 

```
sudo hb-service update-node
ℹ Updating Node.js from v14.17.2 to v14.17.3...
ℹ Updating from NodeSource...
...
nodejs is already the newest version (14.17.2-1nodesource1).
```

So on Raspberry Pi OS, it still only recognizes 14.17.2 as latest available

* Which version of Homebridge/HOOBS do you have?

latest 

* Which platform do you run Homebridge/HOOBS on 

 Raspberry Pi

```console
$ uname -a
Linux rpihp1 5.10.17-v7+ #1421 SMP Thu May 27 13:59:01 BST 2021 armv7l GNU/Linux
```

* Which version of this plugin (homebridge-platform-wemo) do you have? Has the issue started since upgrading from a previous version?

* Which Wemo devices do you have that are causing issues? Please include product models if applicable.

### Please paste any relevant logs below.

```

```

**Comment by bwp91** [comment]
Hi @idcrook 

Strange, on my raspberry pi i was able to update successfully to 14.17.3 (I used `sudo hb-service update-node 14.17.3`)

But you can safely ignore the warning and update the plugin. The plugin will still work correctly.

**Comment by idcrook** [comment]
With the new v3.2.2 plug-in release, the node version requirement was reverted to the widely available 14.17.2 

updating to that version of plugin does not exhibit the required node version warning
